locksmith/src/routes/v2/images.ts

```diff
@@ -38,7 +38,6 @@ const storage =
         },
       })
     : multerS3({
-        // @ts-expect-error Type 'import("/home/unlock/node_modules/@aws-sdk/client-s3/dist-types/S3Client").S3Client' is not assignable to type 'import("/home/unlock/node_modules/@types/multer-s3/node_modules/@aws-sdk/client-s3/dist-types/S3Client").S3Client'.
```

Comment:
that one is actually a fix for https://github.com/unlock-protocol/unlock/pull/13927[)](https://github.com/unlock-protocol/unlock/commit/273dd5fbb55f21cdb9262d7c2e629cfaad6a18bd) it seems